public/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+  <head>
+      <meta charset="utf-8">
+      <title>SpotAVinyl</title>
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
+      <link href="http://fonts.googleapis.com/css?family=Didact+Gothic" rel="stylesheet" />
+      <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery-backstretch/2.0.4/jquery.backstretch.min.js"></script>
+      <link rel="stylesheet" href="vendor/styles/normalize.css">
+      <link rel="stylesheet" href="/styles/base.css">
+      <link rel="stylesheet" href="/styles/layout.css">
+      <link rel="stylesheet" href="/styles/modules.css">
+      <script id="artist-template" type="text/x-handlebars-template">
+          <button class="button" data-id="{{id}}">
+            <div >
+              <div id="pictures"><img src="{{image_url}}" height="100" width="100"></div>
+              <p id="artist-name">{{name}}</p>
+            </div>
+          </button>
+      </script>
+  </head>
+  <body>
+      <div id="header-wrapper">
+          <header>
+              <a href="/index"><img src="/images/spotavinylW.png" /></a>
+              <nav>
+                  <ul>
+                      <li class="tab" id="homepage-tab"><a href="/index">Homepage</a></li>
+                      <li class="tab" id="results-tab"><a href="/results">Results</a></li>
+                      <li class="tab" id="about-tab"><a href="/about">About Us</a></li>
+                  </ul>
+              </nav>
+          </header>
+      </div>
+      <main>
+          <div id="three-column" class="banner">
+              <section id="homepage" class="title">
+                  <h1>Welcome To SpotAVinyl</h1>
+                    <p class="byline">You'll be able to buy vinyl records based on your Spotify tastes.<br /> Click the button below to give us a "spin".</p>
+                    <p><a href="/login" class="button">Log into your Spotify account</a></p>
+              </section>
+              <section id="results" class="title">
+                  <p class="byline">Click on one of your top 10 artists<br>to find their vinyl on eBay.</p>
+                  <div class="artistButtons"></div>
+                  <section id="buy">
+                      <h1>Buy the record here.</h1>
+                      <p class="byline">You can select the best deals below and buy your vinyl.</p>
+                      <div id="ebay"></div>
+                      <a href="#results"><p id="backButton" class="button"> Back to Artists </p></a>
+                  </section>
+              </section>
+              <section id="about" class="title">
+                  <h1>Team LauMadJoe</h1>
+                  <img src="images/laumadjoe.jpg" alt="Photo of Laura, Maddie, and Joe" width="50%" height="50%" />
```

Comment:
Inline styles... If there's a legitimate reason that's fine, but otherwise should be in your CSS.